test_string_calculator.py

```diff
@@ -0,0 +1,43 @@
+from unittest import TestCase
+import unittest
+from string_calculator import StringCalculator
+
+
+class StringCalculatorTest(TestCase):
+    def setUp(self):
+        self.string_calculator = StringCalculator()
+
+    def test_string_calculator_zero(self):
+        test_string = ''
+        expected_number = 0
+        self.assertEqual(self.string_calculator.add(test_string), expected_number)
+    
+    def test_string_calculator_one_value(self):
+        test_string = '3'
+        expected_number = 3
+        self.assertEqual(self.string_calculator.add(test_string), expected_number)
+
+    def test_string_calculator_two_values_delimited(self):
+        test_string = '10,20'
+        expected_number = 30
```

Comment:
Also you can extract the ```response_number = self.string_calculator.add(test_string)```

so that, the test will be more understandable

Reply:
Done.